### ISSUE TITLE
[API, FIXED] Get Player's SkillPoints

### ISSUE THREAD

**Issue by zThana** [elBukkit/MagicLib#6]
Heey,

I'm trying to get the amount of sp a certain player has, but I have no clue how I'd do that using the API.

Could you help me out, please? ;P
- Pr0totype2

**Comment by NathanWolf** [comment]
Does that mean you figured it out? 👍

**Comment by zThana** [comment]
Kinda.. I found the method but it isn't showing me any numbers...

**Comment by NathanWolf** [comment]
Yeah it should just be Mage.getSkillPoints, it would at least show zero. In-game you should see the amount as your XP level while holding a wand.

**Comment by zThana** [comment]
Not true. It doesn't return anything on my scoreboard, just leaves it empty.

**Comment by NathanWolf** [comment]
Not the scoreboard, the XP display- the number above your XP bar (mana bar while holding a wand). Magic doesn't use the scoreboard at all.

It should be there unless you've turned off SP or the SP display in  your configs.

**Comment by zThana** [comment]
I've used the API to receive the amount of SP a player has, but it doesn't show up on the scoreboard, it simply returns nothing.

**Comment by NathanWolf** [comment]
What I am suggesting is that you look at your XP level while holding a wand to make sure you actually have some SP. If it shows zero, then you have zero SP and I'd expect that's what the API will return as well.

The API returns an int, so by "nothing" I'm assuming you mean "0" as it's literally impossible for that method to return nothing.

**Comment by zThana** [comment]
It's actually empty, nothing shows up

**Comment by NathanWolf** [comment]
I think you must be doing something wrong in the scoreboard display then... Mage.getSkillPoints returns an int, there's no way for it to be empty. An int will always be _something_, even if it's zero.

**Comment by zThana** [comment]
I understand that, I'll pm you my scoreboard class via Spigot

**Comment by NathanWolf** [comment]
Sure, I may not be much help there though, I've never used the scoreboard.

I'd suggest just printing the SP value to logs or chat for starters just to make sure you've got what you're expecting.
